src/components/navigation/sw-tabs/sw-tabs.interactive.stories.ts

```diff
@@ -237,32 +236,32 @@ export const VisualTestRenderTabsWithContextMenuBadge = {
         label: 'Context tab test',
         name: 'contextTabTest',
       },
-      ...tabItems.slice(0,8),
+      ...tabItems.slice(0, 8),
       {
         label: 'Item with critical badge',
         name: 'badgeItem',
-        badge: 'critical'
+        badge: 'critical',
       },
       ...tabItems.slice(9),
-    ]
+    ],
   },
   play: async ({ canvasElement }) => {
     const canvas = within(canvasElement);
 
     // wait until tab bar is loaded and context button gets rendered
 
-    await waitUntilRendered(() => document.body.textContent.includes('Context tab test'));
+    await waitUntilRendered(() => document.body.textContent?.includes('Context tab test'));
```

Comment:
That thing just popped into my mind. Testing Library (which is provided through the storybook testing package) already provides such a [function](https://testing-library.com/docs/dom-testing-library/api-async/).

I know that it's be to much for now to change everything again. Though, we could check if that could be an alternative for future tickets. And to get rid of it over time.

Reply:
I know this function. I suggest to do it in another ticket. Also because the "waitFor" methods works a bit differently under the hood. It waits until no error gets thrown anymore. Example for this have to be:
```js
await waitFor(() => {
    const textContentIncludes = document.body.textContent.includes('Context tab test'));

   if (!textContentIncludes) {
       throw new Error('Text content does not include "Context tab test" yet');
   }
})
```

This is a bit more work and doing it in a different ticket would make more sense